src/dndbeyond/content-scripts/character.js

```diff
@@ -527,6 +538,13 @@ function rollAction(paneClass) {
                     damage_types.push("Sneak Attack");
                 }
             }
+            // Wizard: Bladesong
+            if (character.hasClassFeature("Song of Victory") && character.getSetting("wizard-bladesong", false)) {
+                const intelligence = character.getAbility("INT") || {mod: 0};
+                const mod = parseInt(intelligence.mod) || 0;
+                damages.push(String(Math.max(mod, 1)));
+                damage_types.push("Wizard: Bladesong");
```

Comment:
ah, I see the misunderstanding when you told me in discord that sneak attack doesn't say "Rogue: Sneak Attack"... 
The comment I made about the `"${class}: ${feature_name}"` was not about the damage, it was about the settings title :p Here it should just say Bladesong

Reply:
Well make up your mind!

Reply:
My mind's made up, I choose the egg